modules/ROOT/pages/migration-from-6x.adoc

```diff
@@ -3,62 +3,310 @@
+
+[IMPORTANT]
+If customers in-counter table styling issues, they will need to update their CSS to match the **new pattern** produced by {productname}.
 
 [[security]]
 == Security
 
-...
+=== Changed
+
+[[sandbox-iframes-option]]
+==== `sandbox_iframes`
+
+Previous Security penetration testing surfaced a security issue with {productname} loading iframes `unsandboxed` which as a result could allow arbitrary code execution inside {productname} content.
+
+As a result, a configuration option was introduced in {productname} 6.8 to sandbox `(to isolate)` iframes and prevent javascript from executing code inside {productname} documents.
+
+The config option is currently set to `opt in`.
+
+{productname} 7.0, the default value of the `sandbox_iframes` configuration option will be changed from `false` to `true`, making it `opt out` instead of `opt in` for their editors.
+
+[IMPORTANT]
+The premium **Enhanced Media Embed** plugin uses `iframes` extensively and are currently no workarounds available except to keep a record of allowed sources. Customers who want to continue to use **Enhanced Media Embed** plugin in {productname} 7.0 will be required to add the option `sandbox_iframes: false` to their {productname} init configuration.
+
+.Example
+[source, js]
+----
+tinymce.init({
+  selector: "textarea",
+  sandbox_iframes: false,
+});
+----
+
+This follows our policy that we ship the most secure version of {productname} possible, while giving our customers the opportunity to make informed decisions on dialing back protections to a level that fits their use-case.
+
+For more information on the `sandbox_iframes` option see xref:security.adoc#sandbox-iframes[sandbox_iframes].
+
+[[convert-unsafe-embeds-option]]
+==== `convert_unsafe_embeds`
+
+A previous Security penetration test surfaced a security issue with {productname} regarding object tags, which was a legacy method of inserting an external resource, such as a video, a PDF.
+
+{productname} has determined that the object tag is regarded as unsafe, as it may execute javascript inside the {productname} content.
+
+{productname} has a configuration option that converts object tags to modern equivalents such as `<img>, <video>` and `<audio>` tags, such as;
+
+.Example of before/after conversion.
+[source, html]
+----
+<!-- Before Conversion -->
+<object type="video/mp4" data="https://sneak-preview.tiny.cloud/3adc27b5-bb2f-49f0-9ccc-72b7c48313b0/bad.mov"></object>
+
+<!-- After Conversion -->
+<video src="https://sneak-preview.tiny.cloud/3adc27b5-bb2f-49f0-9ccc-72b7c48313b0/bad.mov" controls="controls"></video>
+----
+
+In {productname} 7.0 by default;
+
+* all object tags will be converted by default by loading the editor.
+* the `convert_unsafe_embeds` option default value will change from `false` to `true` making it opt out instead of opt in.
+
+[NOTE]
+To prevent automatic conversion of content, customers are advised to add `convert_unsafe_embeds: false` to their {productname} init configuration.
```

Comment:
@MitchC1999 see suggested update, to match the release notes + include example of the conversion.

```suggestion
In {productname} 6.8.1, xref:content-filtering.adoc#convert-unsafe-embeds[convert_unsafe_embeds] editor option was introduced to allow `+object+` and `+embed+` elements to be converted by default to the correct element, respective of the MIME type, automatically when inserted into the editor.

In {productname} 7.0, the default value for `+convert_unsafe_embeds+` will change from `false` to `true`, meaning that all `+object+` and `+embed+` tags will automatically be converted to different elements when inserted to the editor.

{productname} has a configuration option that converts object tags to modern equivalents such as `<img>, <video>` and `<audio>` tags, such as;

.Example of before/after conversion.
[source, html]
----
<!-- Before Conversion -->
<object type="video/mp4" data="https://sneak-preview.tiny.cloud/3adc27b5-bb2f-49f0-9ccc-72b7c48313b0/bad.mov"></object>

<!-- After Conversion -->
<video src="https://sneak-preview.tiny.cloud/3adc27b5-bb2f-49f0-9ccc-72b7c48313b0/bad.mov" controls="controls"></video>
----
[NOTE]
If this behaviour is undesirable, set `+convert_unsafe_embeds+` to `+false+` in your editor configuration.

For further details on the `+convert_unsafe_embeds+` option, see the xref:content-filtering.adoc#convert-unsafe-embeds[content filtering options], or refer to the xref:security.adoc#convert-unsafe-embeds[security guide], or the link:https://www.tiny.cloud/docs/tinymce/6/6.8.1-release-notes/#new-convert_unsafe_embeds-option-that-controls-whether-object-and-embed-elements-will-be-converted-to-more-restrictive-alternatives-namely-img-for-image-mime-types-video-for-video-mime-types-audio-audio-mime-types-or-iframe-for-other-or-unspecified-mime-types[{productname} 6.8.1 release notes].
```